cdap-docs/examples-manual/source/examples/index.rst

```diff
@@ -23,7 +23,7 @@ Examples
    Spark K-Means <spark-k-means>
    Spark Page Rank <spark-page-rank>
    Web Analytics <web-analytics>
-
+   Stream Conversion <stream-conversion>
```

Comment:
Weird, in my version of index.rst it is above. Not sure why the diff shows it below.